gputop-csv/gputop-csv.js

```diff
@@ -134,6 +136,59 @@ GputopCSV.prototype.list_metric_set_counters = function(metric) {
     stderr_log.log("\nALL: " + all);
 }
 
+var ctx_hw_id_ = [];
+var vgpu_id_ = [];
+var map_vgpuID_hwID = [0];
```

Comment:
These variables should probably by part of the GputopCSV instance.

Reply:
Sorry for my mistake, I have put it into GputopCSV instance.